packages/vitest/src/utils/timers.ts

```diff
@@ -0,0 +1,4 @@
+export const setTimeout = globalThis.setTimeout
```

Comment:
I was thinking about importing them form `timers` module, but then it would be hard to mock them in browser env, I think